ComposeAdvanced/app/src/main/java/com/example/android/wearable/composeadvanced/presentation/navigation/Screen.kt

```diff
@@ -32,4 +32,7 @@ sealed class Screen(
     object Landing : Screen("landing")
     object WatchList : Screen("watchList")
     object WatchDetail : Screen("watchDetail")
+    object ValueDisplay : Screen("valueDisplay")
```

Comment:
The screen name should be more descriptive so devs don't have to read other code to figure out what it does.